content/en/docs/pipelines/sdk/install-sdk.md

```diff
@@ -13,6 +13,11 @@ the Kubeflow Pipelines UI for execution.
 
 All of the SDK's classes and methods are described in the auto-generated [SDK reference docs](https://kubeflow-pipelines.readthedocs.io/en/latest/).
 
+**Note:** If you wan to run your pipelines on a [Tekton](https://tekton.dev/) backend
```

Comment:
```suggestion
**Note:** If you are using [Kubeflow Pipelines with Tekton backend]I(https://github.com/kubeflow/kfp-tekton), 
```

Reply:
@animeshsingh nit:

```suggestion
**Note:** If you are using [Kubeflow Pipelines with the Tekton backend](https://github.com/kubeflow/kfp-tekton),
```

(I think an extra `I` found its way between `]` and `(` 😄 )